Prepare for pypi upload (#2)
hove-io/serpy: README.rst

<http://serpy.readthedocs.org/en/latest/performance.html>`_ faster.

> This is a forked version that include de possibility to remove None field from the output.

Reviewer: ```suggestion
This is a forked version that include the possibility to remove None field from the output.
```

> It is recommanded that you used the upstream version available at  https://github.com/clarkduvall/serpy

Reviewer: ```suggestion
It is recommended to use the upstream version available at https://github.com/clarkduvall/serpy
```
